tests/spec/filter/test_drop_with_expr.py

```diff
@@ -1,37 +1,27 @@
-from databuilder.query_language import Dataset, IdSeries, IntSeries, build_event_table
+from ..tables import e
 
-from ..helpers import transpose
+table_data = {
+    e: """
+          |  i1 |  i2
+        --+-----+-----
+        1 | 101 | 111
+        1 | 102 | 112
+        1 | 103 | 113
+        2 | 201 | 211
+        2 | 202 | 212
+        2 | 203 | 213
+        3 | 301 |
+    """,
+}
 
-e = build_event_table(
-    "e",
-    {
-        "patient_id": IdSeries,
-        "i1": IntSeries,
-        "i2": IntSeries,
-    },
-)
 
-
-def test_drop_with_expr(in_memory_engine):
-    in_memory_engine.setup(
+def test_drop_with_expr(spec_test):
+    spec_test(
+        table_data,
+        e.drop((e.i1 + e.i2) < 413).i1.sum_for_patient(),
```

Comment:
This expression seems more complex than is necessary. Could we do without the sum?